rllib/utils/replay_buffers/reservoir_replay_buffer.py

```diff
@@ -122,8 +123,11 @@ def set_state(self, state: Dict[str, Any]) -> None:
 
         Args:
             state: The new state to set this buffer. Can be
-                obtained by calling `self.get_state()`.
+                    obtained by calling `self.get_state()`.
```

Comment:
Why the extra 4-spaces indent? Should be only indented once, no?

Reply:
Because it's included in sphinx now and sphinx complains otherwise. The other solution is -4 spaces indent.